packages/fluentui/docs/src/components/ComponentDoc/ComponentDocLinks.tsx

```diff
@@ -25,7 +25,7 @@ export default class ComponentDocLinks extends React.PureComponent<any, any> {
         }}
       >
         <>
-          <Icon name="github" />
+          <Image src="public/images/github.png" width="16px" height="16px" />
```

Comment:
This looks like a good case to create a custom icon with the factory 🤔

Reply:
Not sure if I can just copy svg from other libraries.. If you can think of some source let me know..

Reply:
Probably it's not safe due licensing issues 🤔